src/components/photo_gallery/photo_gallery.js

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback, useState } from "react";
+import ReactBnbGallery from "react-bnb-gallery";
+import classNames from "classnames";
+
+import "react-bnb-gallery/dist/style.css";
+import styles from "./photo_gallery.module.css";
+
+export default function PhotoGallery({ photos }) {
+  const [isGalleryShown, setIsGalleryShown] = useState(false);
+
+  const toggleGallery = useCallback(() => {
+    setIsGalleryShown(!isGalleryShown);
+  }, [isGalleryShown]);
+
+  const parsePhotos = photos.map((photo) => {
+    return { photo: photo.url };
+  });
+
+  if (!photos || !photos.length) {
```

Comment:
you have check for it in parent component - so either remove it here or inside of hotel_page.js